### ISSUE TITLE
alt-slash (console) hangs

### ISSUE THREAD

**Issue by ccoupe** [Shoes3/exe-shoes#2]
Bringing up the Shoes console with an alt-slash in the packaged app hangs the app. Odds are high my min-shoes.rb isn't quite right.

**Comment by dredknight** [comment]
Is it be possible to remove all shortcuts for the mini shoes? 
This is not meant for debugging purposes but to be the mini-est possible version and to be hidden from the user.

**Comment by ccoupe** [comment]
just modify min-shoes.rb  to not open the log window.  Now, passing that as an argument might be tricky, but min-shoes.rb is meant to be understood and tailored for the developer's  particular needs.

**Comment by dredknight** [comment]
Thank you @ccoupe. I have not yet seen what is there in mini-shoes so this is a good moment to see :).

**Comment by dredknight** [comment]
I would have to pass on this one as mini-shoes content offers a bunch of methods and classes I dont comprehend how they work. Anyways I will leave this for later. There are more relevant stuff to do :).

**Comment by ccoupe** [comment]
Yes, its make little sense, even to me. min-shoes is responding to events  and one of them is the translated call to show the console/log  Notice  look at how it traps the irb key and does nothing.

**Comment by dredknight** [comment]
This issue can be closed as it occurs no more!
